bitcoind-regtest/test/Bitcoin/Core/Test/Wallet.hs

```diff
@@ -0,0 +1,323 @@
+{-# LANGUAGE NumericUnderscores #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TupleSections #-}
+{-# OPTIONS_GHC -Wno-type-defaults #-}
```

Comment:
What does this do?

Reply:
Happily, it does nothing now.  I removed it.